makefile.shade

```diff
@@ -20,7 +20,9 @@ functions
         const string UniverseCommitsFileName = "commits-universe";
 
         private static bool Quiet { get; set; }
-
+        private static bool UseGateBranch { get; set; }
+        private const string GateBranch = "gateDev";
```

Comment:
Can you make this an environment variable instead? I'm worried that once we create that branch someone will forget to delete it and it might sync again to it.